components/identifier-generator/front/src/feature/components/LabelTranslations.tsx

```diff
@@ -31,6 +32,7 @@ const LabelTranslations: React.FC<LabelTranslationsProps> = ({labelCollection, o
         </SectionTitle.Title>
       </SectionTitle>
       <Styled.FormContainer>
+        {isLoading && !error && <LabelTranslationsSkeleton></LabelTranslationsSkeleton>}
```

Comment:
There is no need to have opening and closing tags when there is not children

```suggestion
        {isLoading && !error && <LabelTranslationsSkeleton/>}
```